<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Visualization3D</class>
 <widget class="QMainWindow" name="Visualization3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Brain Visualization</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QVTKOpenGLWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>721</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="thresholdBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Threshold</string>
    </property>
    <widget class="QLabel" name="thresholdLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Change threshold and accept:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="acceptBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTextEdit" name="thresholdTextEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="visualizationBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Visualization</string>
    </property>
    <widget class="QPushButton" name="visualizeBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualize 3D</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="planeGroupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Cutting plane option</string>
    </property>
    <widget class="QRadioButton" name="cutEnableRadioBtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="processGroupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>580</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Process state</string>
    </property>
    <widget class="QLabel" name="processDescLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="shrinkDataGroupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Shrinking data</string>
    </property>
    <widget class="QSlider" name="shrinkSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="shrinkingLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>* Shrink data to increase efficiency</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
